src/App.css

```diff
@@ -1,29 +1,79 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+#root {
+  background-color: #1b1834;
+}
```

Comment:
- [ ] As requested by the previous reviewer, Please make sure that your project design matches the [Zeplin](https://zeplin.io/) design. It's part of the requirement that the design matches the Zipline design including all elements + the same styling.

![image](https://user-images.githubusercontent.com/83803414/209447173-7138dbb5-5d56-4edb-bb86-bb5e52412541.png)